include/ensmallen_bits/sgdr/sgdr.hpp

```diff
@@ -71,11 +72,13 @@ class SGDR
    *        parameters.
    * @param resetPolicy If true, parameters are reset before every Optimize
    *        call; otherwise, their values are retained.
+   * @param stepSizeMin Final step size before each restart
    */
   SGDR(const size_t epochRestart = 50,
        const double multFactor = 2.0,
        const size_t batchSize = 1000,
-       const double stepSize = 0.01,
+       const double stepSizeMax = 0.01,
+       const double stepSizeMin = 0.005,
```

Comment:
So, unfortunately, this is a reverse-compatibility-breaking change.  We should add a deprecated constructor (don't worry about marking it as deprecated, we can do that in #113) that matches the old format if needed.

```
SGDR(const size_t epochRestart, const double multFactor, const size_t batchSize, const double stepSize,
    const size_t maxIterations, const double tolerance = 1e-5, const bool shuffle = true, 
    const UpdatePolicyType& updatePolicy = UpdatePolicyType(), const bool resetPolicy = true);
```

(We can't use default constructors before the `maxIterations` bit otherwise there will be ambiguity in the constructors.)

In this deprecated constructor we could just take `stepSizeMin = 0.5 * stepSizeMax` or something.